docs/get-started/quickstarts/detect-people.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Detect People with a Webcam"
+linkTitle: "Detect people with a webcam"
+weight: 50
+type: "docs"
+tags: ["vision", "data", "services", "quickstart", "ml", "camera", "webcam"]
+no_list: true
+description: "Use Viam's machine learning capabilities to deploy a person detector ML model to your machine."
+images: ["/platform/ml.svg"]
+imageAlt: "Machine Learning"
+---
+
+Follow this guide to detect people with a webcam on the Viam platform, using a [pre trained ML model from the registry](https://app.viam.com/ml-model/ml-models-scuttle/people).
+
+## Requirements
+
+- A computer: a development machine like your laptop and optionally a single-board computer like the Raspberry Pi
+- A webcam: this could be the webcam on your laptop or any other webcam you can connect to your computer
+- An account on the [Viam app](https://app.viam.com)
+
+First, follow the [instructions to create a new machine](/cloud/machines/#add-a-new-machine) representing your computer.
+Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer and connect it to the Viam app.
+
+You need to configure the following {{< glossary_tooltip term_id="resource" text="resources" >}} to detect people with a webcam:
+
+1. A [`webcam` camera](/components/camera/webcam/)
+2. An [`tflite_cpu` ML model service](/services/ml/deploy/tflite_cpu/)
+3. An [`mlmodel` computer vision service](/services/vision/mlmodel/)
+
+Follow these instructions to configure your machine and test detections:
+
+{{%expand "Step 1: Configure your webcam" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Select the `camera` type, then select the `webcam` model.
+Enter a name or use the suggested name for your camera and click **Create**.
+
+Click the **Save** button in the top right corner of the page to save your config.
```

Comment:
Nit: full word unless referring to UI element or code verbatim. Unpopular opinion though, feel free to overrule me if you strongly disagree 👍 

```suggestion
Click the **Save** button in the top right corner of the page to save your configuration.
```